go.mod

```diff
@@ -6,20 +6,20 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gonum/graph v0.0.0-20170401004347-50b27dea7ebb
-	github.com/openshift/api v0.0.0-20200701144905-de5b010b2b38
-	github.com/openshift/build-machinery-go v0.0.0-20200512074546-3744767c4131
-	github.com/openshift/client-go v0.0.0-20200521150516-05eb9880269c
-	github.com/openshift/library-go v0.0.0-20200720190023-e1f4b4f4d448
+	github.com/openshift/api master
```

Comment:
I don't think this should be separate, just run `go mod tidy && go mod vendor` and commit the result as bump or you are changing the same lines twice in the same PR

Reply:
yeah, go mod sucks. but this doesn't give any guarantee, maybe you have amended after this commit and didn't propagate it back. Comment would probably have the same weight, but feel free to leave here.